eppoclient/assignmentlogger.go

```diff
@@ -8,6 +8,8 @@ type IAssignmentLogger interface {
 
 type AssignmentEvent struct {
 	Experiment        string
+	FeatureFlag       string
+	Allocation        string
```

Comment:
If these are required, would this break the client's integrations until they update their logger? If so, is there a way to make these new fields optional?

Reply:
Here is example usage - https://github.com/Eppo-exp/sdk-examples/blob/main/go-server-example/main.go#L20-L23

The client is not yet using these new fields

Reply:
oh right, because the client is importing `AssignmentEvent` so it will be updated on their end too 🤦‍♂️ 

Carry on!